diff -urN gnome-crystal-0.6.1.orig/gcrystal.desktop.in.in gnome-crystal-0.6.1/gcrystal.desktop.in.in
--- gnome-crystal-0.6.1.orig/gcrystal.desktop.in.in	2003-03-02 11:46:35.000000000 +0100
+++ gnome-crystal-0.6.1/gcrystal.desktop.in.in	2005-05-25 14:44:49.000000000 +0200
@@ -1,9 +1,10 @@
 [Desktop Entry]
+Encoding=UTF-8
 _Name=Crystalline structures viewer
 _Comment=Edit and display crystalline structures
 Exec=@BINDIR@/gcrystal
 Icon=gnome-crystal.png
-Terminal=0
+Terminal=false
 Type=Application
 DocPath=gcrystal/gcrystal.xml
-Categories=Application;Utility;ChemistryEditor
+Categories=GTK;GNOME;Education;Science;Chemistry;
